scripts/RNA_analysis/make_complexity_plots.py

```diff
@@ -46,11 +48,19 @@
 #SBATCH {5}
 
 module load bioinfo-tools
+module load {6}
 module load {3}
 module load {2}
 
 cd {4}
+""".format(name, mail, preseq, bam, path, extra_arg, bed)
+
+if len(sys.argv)==6:
+	print >>f, """bedtools bamtobed -i tophat_out_{0}/accepted_hits_sorted_{0}.bam | sort -k 1,1 -k 2,2n -k 6,6 >tophat_out_{0}/accepted_hits_sorted_preseq_{0}.bed
```

Comment:
I don't quite get this - why are you converting to bed format? You can supply `preseq` with the `-B` option and it should understand BAM files. Also, the BAM file is already sorted, so why do you need to sort it again?